src/cache.js

```diff
@@ -99,7 +99,9 @@ class cache {
     let data = await this.get(key, toJSON)
     if (!data) { // data is empty
       data = await caller(...callerParams)
-      this.set(key, data, time) // async set
+      if (data) {
```

Comment:
If this judge is necessary? Maybe we already do this at line 100.

Reply:
The data was be reassigned by the func `caller`. `caller` is a func that can be filled as custom.
This is a usage :
```
Cache.remeber(key, time, callerFunc, [callerParams,[toJSON]])
Cache.rember(
  'test', // Redis Key
  60, // 60 seconds
  async (a, b, c) => { // a func was used to fetch data
      return a + b + c
   },
   ['a', 'b', 'c'],  // the array will be filled as the params of the caller func
   true // the result data(fetched by Redis Cache) will be parsed by `JSON.parse`
```